src/solver/optimisation/opt_gestion_des_couts_cas_lineaire.cpp

```diff
@@ -249,6 +249,12 @@ void OPT_InitialiserLesCoutsLineaire(PROBLEME_HEBDO* ProblemeHebdo,
             {
                 ProblemeAResoudre->CoutLineaire[Var]
                   = ProblemeHebdo->CoutDeDefaillancePositive[Pays];
+            
+            Var = CorrespondanceVarNativesVarOptim->IndexOfVariableDomesticEnergyNotServed[Pays];
+            if (Var >= 0 && Var < ProblemeAResoudre->NombreDeVariables)
+            {
+                ProblemeAResoudre->CoutLineaire[Var]
+                  = 1.0;
```

Comment:
Thank you !!
I tried to track it down, but I don't think GROSSES_VARIABLES is ever set to OUI_ANTARES. I believe this to be a security for big variables with high prices, in order not to overflow the solver. Since our dummy variable only has a cost of 1$/MW it should be fine.